### PR TITLE
allow branch override in cluster launch files

### PR DESCRIPTION
**Pull request by tomsilver**
because it's currently annoying to test new config files before merging them (because the new config files will have the 'master' branch in them, but they're not yet on master themselves)

### CODE REVIEW

File: scripts/local/launch.py [hunk 1]
```diff
@@ -18,8 +18,13 @@ def _main() -> None:
     # Set up argparse.
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", required=True, type=str)
+    # Optionally override the branch specified in the CFG.
```
Comment (ronuchit) [inline scripts/local/launch.py:21]: i would suggest we don't use "CFG" here, this is a different config file than settings.py right?

**Review by ronuchit — APPROVED** [review]
i'm wondering if it would make sense to just not have a "branch" in these config files and instead either (1) have it always default to master (without needing to type it in every config file) or (2) force the user to pass in a `--branch` into the command line? either one sounds good to me, but as it stands, i don't see why we'd ever *not* have "BRANCH: master" in the config, especially after this PR is merged

**Comment by tomsilver** [comment]
okay good point, let's do the first option -- defaulting to master but allowing command line overrides

**Comment by tomsilver** [comment]
gonna do this differently and also some other time

**Pull request closed without merging**